src/components/folder/common/FolderProfile.js

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect, useState } from 'react';
+
+const FolderProfile = () => {
+  const [profileData, setProfileData] = useState(null);
+
+  useEffect(() => {
+    fetch('https://bootcamp-api.codeit.kr/api/users/1')
+      .then((response) => response.json())
+      .then((data) => {
+        setProfileData(data.data[0]);
```

Comment:
```suggestion
      .then(({ data }) => {
        setProfileData(data[0]);
```
이렇게 사용해도 동일하게 동작합니다.